sbi/analysis/plot.py

```diff
@@ -693,17 +634,17 @@ def pairplot(
     ] = None,
     limits: Optional[Union[List, torch.Tensor]] = None,
     subset: Optional[List[int]] = None,
-    upper: Optional[Union[List[Optional[str]], str]] = "hist",
-    lower: Optional[Union[List[Optional[str]], str]] = None,
-    diag: Optional[Union[List[Optional[str]], str]] = "hist",
+    upper: Optional[Union[List[Optional[UpperLiteral]], UpperLiteral]] = "hist",
```

Comment:
general comment for the position of `pairplot` in this file: I always need to scroll a lot to find this function when I want to look up something. Of course, I could just search it, but I think it would be handy to just have it at the top of the file because it's the central plotting function.